client/src/contexts/Okteto.context.tsx

```diff
@@ -33,16 +34,19 @@ type OktetoProviderProps = {
 const Okteto = createContext<OktetoStore | null>(null);
 
 const CONTEXT_POLLING_INTERVAL = 3000;
+const STATUS_POLLING_INTERVAL = 3000;
 export const defaultContextName = 'https://cloud.okteto.com';
 
 const OktetoProvider = ({ children } : OktetoProviderProps) => {
   const [currentContext, setCurrentContext] = useState<OktetoContext | null>(null);
   const [contextList, setContextList] = useState<OktetoContextList>([]);
   const [environment, setEnvironment] = useState<OktetoEnvironment | null>(null);
+  const [status, setStatus] = useState<OktetoStatus | null>(null);
```

Comment:
```suggestion
  const [status, setStatus] = useState<OktetoStatus>('pending');
```

Reply:
We have to set it back to it's initial value also in the `stopEnvironment` function, otherwise we will initially carry the last status to the next environment launched.